src/foremast/templates/configs/configs.json.j2

```diff
@@ -5,6 +5,7 @@
         "approval_timeout": null,
         "archaius_enabled": false,
         "canary": false,
+        "custom_tags": {},
```

Comment:
One way to add tags for all the AWS pipeline types is great 👍

Reply:
Could extend to GCP too if ever needed. :)